README.md

```diff
@@ -43,68 +65,34 @@ For more details see [#405](https://github.com/watson-developer-cloud/python-sdk
 
 The [examples][examples] folder has basic and advanced examples.
 
-## Getting the Service Credentials
-
-Service credentials are required to access the APIs.
-
-If you run your app in IBM Cloud, you don't need to specify the username and password or IAM API key (`apikey`). In that case, the SDK uses the `VCAP_SERVICES` environment variable to load the credentials.
-To run locally or outside of IBM Cloud you need the `username` and `password` credentials or IAM API key (`apikey`) for each service. (Service credentials are different from your IBM Cloud account email and password.)
-
-To create an instance of the service:
-
-1. Log in to [IBM Cloud][ibm_cloud].
-1. Create an instance of the service:
-   1. Click on **Create Resource**.
-   1. In the IBM Cloud **Catalog**, select the Watson service you want to use. For example, select the Conversation service.
-   1. Type a unique name for the service instance in the **Service name** field. For example, type `my-service-name`. Leave the default values for the other options.
-   1. Click **Create**.
-
 ## Authentication
-To get your service credentials:
 
-Copy your credentials from the **Manage** page. To find the Service details page for an existing service, navigate to your [IBM Cloud][ibm_cloud] dashboard and click the service name.
+Watson services are migrating to token-based Identity and Access Management (IAM) authentication.
 
-1. On the **Manage** page, you will see a **Credentials** pane
-1. Depending on the service you will see use either:
-* 2.a: `username`, `password`, and `url`(optional) or `apikey`(for Visual Recognition).
-* 2.b: `apikey` which is the value for parameter `iam_api_key` when initializing the constructor.
+- With some service instances, you authenticate to the API by using **[IAM](#iam)**.
+- In other instances, you authenticate by providing the **[username and password](#username-and-password)** for the service instance.
+- Visual Recognition uses a form of [API key](#api-key) only with instances created before May 23, 2018. Newer instances of Visual Recognition use IAM.
 
-### Username and Password
-```python
-from watson_developer_cloud import DiscoveryV1
-# In the constructor
-discovery = DiscoveryV1(version='2017-10-16', username='<username>', password='<password>')
-```
-
-```python
-# After instantiation
-discovery = DiscoveryV1(version='2017-10-16')
-discovery.set_username_and_password('<username>', '<password>')
-```
+**Tip**: To find out which authentication to use, view the service credentials by clicking the service instance on the IBM Cloud [Dashboard](https://console.bluemix.net/dashboard/apps?watson).
```

Comment:
https://console.bluemix.net/dashboard/apps?category=watson

Reply:
This is the link I got from Sabtain and German that's supposed to work. Did it change? Posted comment in slack